repo-transfer.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Transfer a repository to the W3C GitHub organization</title>
+    <link rel="stylesheet" href="css/wgio.css">
+  </head>
+  <body>
+    <header>
+      <h1>Transfer a repository to the W3C GitHub organization</h1>
+    </header>
+<nav>
+  <a href="/">Home</a>
+  •
+  <a href="https://github.com/w3c/">Repositories</a>
+  •
+  <a href="https://help.github.com/">GitHub Help</a>
+</nav>
+
+<main>
+  <p>This document describes how to transfer a repository to the <a href="https://github.com/w3c/">W3C GitHub organization</a>.</p>
+  <p>General information about repository transfer on GitHub is available <a href="https://docs.github.com/en/github/administering-a-repository/transferring-a-repository">here</a>.</p>
```

Comment:
Rather than "here" as a link, suggest: "See also <a>general information about GitHub repo transfer</a>.

Reply:
Addressed in https://github.com/w3c/w3c.github.io/pull/96/commits/e43f69dd0ec64f42ba31309b109a5c716ba093ff